app/views.py

```diff
@@ -22,14 +22,23 @@
 
 
 database = DatabaseConnection()
-order = Orders()
-menu = Menu()
+# order = Orders()
+# menu = Menu()
 user = User()
 
 database.create_user_table()
 database.create_menu_list()
 cur = database.cursor
 
+#New users can create accounts
+@app.route('/api/v1/auth/signup', methods = ['POST'])
+def create_new_user():
+    data = request.get_json()
+    userId = data['userId']
```

Comment:
I think here you would want your user id to be automatically generated because the user can't provide his id.

Reply:
i have that blocker at the moment hoping to resolve it shortly. thank you

Reply:
You may as well help suggest how fast i can do it